applications/nrf_desktop/doc/dfu.rst

```diff
@@ -226,7 +237,7 @@ Writing data to non-volatile memory
 ===================================
 
 The image data that is received from the host is initially buffered in the RAM.
-Writing the data to non-volatile memory is triggered when the host performs the fetch operation on the ``sync`` option.
+Writing the data to non-volatile memory, such as flash or MRAM, is triggered when the host performs the fetch operation on the ``sync`` option.
```

Comment:
beside flash and MRAM there is also RRAM on nrf54l. do we have to explain what types of nvm there are here?